src/App.js

```diff
@@ -1,7 +1,9 @@
 import React, { useState, useEffect } from 'react';
 import { getAllData } from './util/index';
 import { createServer } from 'miragejs';
-// import SignIn from './components/SignIn';
+import AlbumGrid from './components/AlbumGrid';
+import './App.css';
```

Comment:
This is fine for now; but it's preferable to not have all the CSS for an application in one file. So since currently the styles in this file are mainly for the `AlbumGrid` (and its children) my suggestion would be to rename it `AlbumGrid.css` , and instead of importing at the App.js level, import it in `AlbumGrid.js`